sfge/src/main/java/com/salesforce/apex/jorje/AbstractJorjeNode.java

```diff
@@ -114,11 +119,24 @@ public final void setMetaInformation(PositionInformation positionInformation) {
         endLine = positionInformation.getLineNumber(location.getEndIndex());
         beginColumn = location.getColumn();
 
+        computeChildIndices(0, false);
+    }
+
+    /**
+     * Computes (or recomputes) the indices associated with each of this node's children.
+     *
+     * @param startOffset Offset applied to the indices. Useful if synthetic vertices are inserted
+     *     at the head of the list.
+     * @param expectingSubsequentChildren If true, the final child's {@link #lastChild} property is
+     *     set to false.
+     */
+    @Override
+    public final void computeChildIndices(int startOffset, boolean expectingSubsequentChildren) {
```

Comment:
Pulling this part into its own method allows child indices to be recomputed on the fly if need be.